lib/recurly/adjustment.ex

```diff
@@ -48,6 +48,30 @@ defmodule Recurly.Adjustment do
   end
 
   @doc """
+  Lists the adjustments from a given account.
+
+  ## Parameters
+
+  - `account_code` String account_code
+
+  ## Examples
+
+  ```
+  alias Recurly.NotFoundError
+
+  case Recurly.Adjustment.list("account_code") do
+    {:ok, adjustment} ->
+      # Found the adjustment
+    {:error, %NotFoundError{}} ->
+      # 404 adjustment was not found
+  end
+  ```
+  """
+  def list(account_code, options \\ []) do
```

Comment:
Also we changed from using `list` to `stream`. See: https://github.com/bhelx/recurly-client-elixir/blob/master/lib/recurly/subscription.ex#L49 Let's change this to a `stream` function. I want encourage use of the stream interface and if someone wants to do pagination they can use `Recurly.Resource.list/3`which now returns pages.